code/datums/datacore.dm

```diff
@@ -415,7 +416,6 @@
 	var/datum/db_record/bank_record = null
 	var/target_byond_key = null
 	var/issuer_byond_key = null
-	var/approver_byond_key = null
```

Comment:
What's the reason for removing this? Approver could still be different to issuer

Reply:
The var wasn't used anywhere, and the logging uses usr. Didn't think there was a particular need for a var, but putting it back